### ISSUE TITLE
Show block data

### ISSUE THREAD

**Issue by cpacia** [OpenBazaar/openbazaar-desktop#1627]
The show block data button from advanced settings needs to get wired up to the new api format. It's probably OK to show all coins in the same view.

**Comment by jjeffryes** [comment]
For reference, the data from wallet/status endpoint will come in like this:
```
{
    "BCH": {
        "bestHash": "000000000000000000d13af268ef2749c65f8a0de621bbef41fcffca65161413",
        "height": 558594
    },
    "BTC": {
        "bestHash": "0000000000000000000000000000000000000000000000000000000000000000",
        "height": 0
    },
    "LTC": {
        "bestHash": "d580f54267b82da648a09ea33b5f384aea994d1984d1fb6ee37d62a42e50e801",
        "height": 1534448
    },
    "ZEC": {
        "bestHash": "0000000000000000000000000000000000000000000000000000000000000000",
        "height": 0
    }
}
```

In the data above, the BTC and ZEC servers are down. Instead of showing a bunch of zeroes, we should probably show some sort of error message indicating the data isn't available.

**Comment by jjeffryes** [comment]
This was closed in #1631
